model/shared/src/main/scala/explore/model/ElevationRange.scala

```diff
@@ -4,17 +4,39 @@
 package explore.model
 
 import cats._
+import cats.syntax.all._
 import eu.timepit.refined.api.Refined
 import eu.timepit.refined.numeric.Interval
 import io.circe.Decoder
 import io.circe.Encoder
+import io.circe._
 import io.circe.generic.semiauto._
 import io.circe.refined._
 import lucuma.core.optics.SplitEpi
 import monocle.Prism
 import monocle.macros.GenPrism
 import monocle.macros.Lenses
 
+private sealed abstract class ElevationRangeType(val typeName: String) { self =>
```

Comment:
`ConstraintSetModel` was removed in favor of the generated classes from the query. We still use these classes to model ElevationRange, so decoders were moved here.